consensus/core/src/metrics.rs

```diff
@@ -302,6 +303,12 @@ impl NodeMetrics {
                 "Designates whether the synchronizer scheduler task to fetch blocks is currently running",
                 registry,
             ).unwrap(),
+            fetch_blocks_scheduler_skipped: register_int_counter_vec_with_registry!(
+                "fetch_blocks_scheduler_skipped",
+                "Number of times the scheduler skipped fetching blocks",
```

Comment:
I think it is not very clear which component this metric should be used. I may send a followup PR to rename.